src/ucs/profile/profile_defs.h

```diff
@@ -14,10 +14,13 @@
 
 BEGIN_C_DECLS
 
-#define UCS_PROFILE_STACK_MAX       64
-#define UCS_PROFILE_FILE_VERSION    2u
-#define UCS_PROFILE_LOC_ID_UNKNOWN  -1
-#define UCS_PROFILE_LOC_ID_DISABLED 0
+#define UCS_PROFILE_ENV_VARIABLES_MIN_VERSION           3u
+
+#define UCS_PROFILE_STACK_MAX                           64
+#define UCS_PROFILE_FILE_VERSION                        UCS_PROFILE_ENV_VARIABLES_MIN_VERSION
```

Comment:
Maybe use here directly 3 instead. The reason is to explicitly decouple it from the "env_vars" version.

Suppose in the future we have feature_{x,y}_version=4.
Then what do we put here?